Gemfile.lock

```diff
@@ -580,6 +580,3 @@ DEPENDENCIES
   weibo_2!
   wunderground (~> 1.2.0)
   xmpp4r (~> 0.5.6)
-
```

Comment:
How do I stop this from being removed?  I updated bundler and now `bundler --version` reports `Bundler version 1.10.6`, but whenever I run `bundle`, this happens.  Oddly, even editing the Gemfile.lock and adding it back in seems to be unable to save the change, like some background process keeps removing it.